app/views/user/trips/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<h1 class="text-center1">Trip Preferences</h1>
+<%= form_with model: @trip, url: user_trips_path do |form| %>
+  <%= form.label :location %>
+  <%= form.text_field :location %>
+  <br>
+  <%= form.label :start_date %>
+  <%= form.date_field :start_date %>
+  <br>
+  <%= form.label :end_date %>
+  <%= form.date_field :end_date %>
+  <section class="cuisine_options">
+    <h1 class="text-center1">Cuisine Options</h1>
+    <%= form.check_box "indian" %>
+    <%= form.label "Indian" %>
+    <%= form.check_box "mexican" %>
+    <%= form.label "Mexican" %>
+    <%= form.check_box "pizza" %>
+    <%= form.label "Pizza" %>
+    <br>
+    <%= form.check_box "italian" %>
+    <%= form.label "Italian" %>
+    <%= form.check_box "chinese" %>
+    <%= form.label "Chinese" %>
+    <%= form.check_box "american" %>
+    <%= form.label "American" %>
+  </section>
```

Comment:
we may want to add more options later. we don't have to if first priority is mvp and function

Reply:
+1, In the future some type of dynamic list of options from our api would make sense. Not required for MVP